added section hero

DennisUser1 opened the pull request:
added section hero - markup + styles mobiles with 320px, tablet with 768px, desktop with 1440px + created filder img under - "hero" section, img has the format "webp", NEEDS TO BE REPLACED WITH FORMAT "png" + add animation to text

DennisUser1 committed:
added section hero

added section hero - markup + styles mobiles with 320px, tablet with 768px, desktop with 1440px + created filder img under - "hero" section, img has the format "webp", NEEDS TO BE REPLACED WITH FORMAT "png" + add animation to text

sukrutdemasek reviewed (changes requested):
Секція зроблена добре, але треба зробити пару правок:
1. Приберіть паддінг справа в десктопній версії, через нього все зсувається
2. У вас кнопки працюють лише при натисненні на сам текст, але не на всю кнопку, це також треба змінити (зазвичай робиться через width:100%; height:100%;

Більше проблем не виявлено

DennisUser1 committed:
fix button click, horizontal scroll

Fix button click, removed horizontal scroll + added animation text ( connected new script for animation in the file 'index.html')+ change img format "png"

sukrutdemasek reviewed (changes requested):
Зробіть указані зміни:
полагодьте паддінги та виправте зсув фонового зображення

DennisUser1 committed:
fix padding container

fix padding container mobile 16px, tablet, desktop - 32px - in this case, the 'hero-box' class + fix property background-size tablet + desktop
